timecog/timecog.py

```diff
@@ -196,7 +196,7 @@ async def remindme(self, ctx, *, time):
         async with self.config.user(ctx.author).reminders() as rms:
             rms.append((rmtime.timestamp(), input, -1, 0))
 
-        response = "I will tell you " + format_rm_time(rmtime, input, user_timezone)
+        response = "I will tell you " + format_rm_time(rmtime, input, user_timezone, await self.config.user(ctx.author).show_tz())
```

Comment:
instead of calling `await self.config.user(ctx.author).show_tz()` I might fetch this once at the start as `user_show_tz_preference` or something

more for readability than performance